src/views/List.jsx

```diff
@@ -1,13 +1,50 @@
+import { useState } from 'react';
+
 import { ListItem } from '../components';
 
 export function List({ data }) {
+	const [searchTerm, setSearchTerm] = useState('');
+
+	const filterList = (list) => {
+		return list.filter(({ name }) => {
+			return name
+				.toLowerCase()
+				.trim()
+				.replace(/\s+/g, ' ')
```

Comment:
This is something we can talk about on Sunday, but would you be able to explain this regex and why it was needed?

Reply:
Originally we were only handling single extra spaces in the search term with the `trim` method. Helen had the idea of handling more than one extra space character

I was able to implement that with a replace method and RegEx - 
`\s` - selects white space characters 
`+` - extends the whitespace character selection to all multiple occurrences in a row
`/g` - searches entire string for occurrences 

all matches are replaced with a `' '` - single space

I assumed I could then remove the `trim` methods, but after trying that out, it seemed I needed both `trim` and the `replace` to handle both single and multiple extra spaces